staging/src/k8s.io/api/core/v1/well_known_labels.go

```diff
@@ -39,4 +39,7 @@ const (
 	// controllers and kube-proxy to check if the Endpoint objects should be replicated when
 	// using Headless Services
 	IsHeadlessService = "service.kubernetes.io/headless"
+
+	// LabelServiceName is used to indicate the name of a Kubernetes service.
+	LabelServiceName = "kubernetes.io/service-name"
```

Comment:
if this label is being applied to endpoint slices, should it go in the alpha endpoints package? this is a very generic label to go in the core API package (and raises the question what it is selecting and what objects it is placed on)

Reply:
This value is also used by cloud load balancers for [GCP](https://github.com/kubernetes/kubernetes/blob/master/staging/src/k8s.io/legacy-cloud-providers/gce/gce_loadbalancer_naming.go#L90) and [AWS](https://github.com/kubernetes/kubernetes/blob/master/staging/src/k8s.io/legacy-cloud-providers/aws/aws.go#L91). Although those references are currently hardcoded in those packages individually, it made me think this label was not EndpointSlice specific and should belong closer to the Service itself (`core/v1`). Happy to move this over to discovery package though, wasn't sure which made the most sense.

Reply:
If there is not enough justification to move this label into core/v1, probably just make it exposed in the discovery v1alpha1 package. 

I do not see much value refactoring legacy cloud provider at this point.

Reply:
:+1: moved to discovery/v1alpha1